packages/website/bin/index.js

```diff
@@ -19,17 +19,22 @@ if (mainOptions.command === undefined) {
   throw new Error(`No command supplied`);
 }
 
+const handleError = err => {
+  console.error(err);
```

Comment:
Does this error come back as a string or an object?

Reply:
It's an instance of Error with a string message